app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from typing import Union
+from app.cafe import Cafe
+from app.errors import (
+    NotWearingMaskError,
+    VaccineError
+)
+
+
+def go_to_cafe(friends: list[dict[str, Union[str, bool]]], cafe: Cafe) -> str:
```

Comment:
Code Style: Please use type annotations for your function parameters. It's a good practice to use them as it makes your code more understandable. Here is how you can do it:
python
def go_to_cafe(friends: List[Dict[str, Union[str, bool]]], cafe: Cafe) -> str:
    ...
```